package.json

```diff
@@ -42,42 +41,42 @@
     "release:major": "npm run release -- --release-as major"
   },
   "peerDependencies": {
-    "@angular/animations": "~9.1.12",
-    "@angular/common": "~9.1.12",
-    "@angular/compiler": "~9.1.12",
-    "@angular/core": "~9.1.12",
-    "@angular/forms": "~9.1.12",
-    "@angular/platform-browser": "~9.1.12",
-    "@angular/platform-browser-dynamic": "~9.1.12",
-    "@angular/router": "~9.1.12",
-    "rxjs": "~6.5.4",
+    "@angular/animations": "^10.2.2",
```

Comment:
it will broke angular 9 installs
https://nodejs.org/uk/blog/npm/peer-dependencies/#using-peer-dependencies
peer deps are not installed during setup if matched with parent package.json
please change to more then 9, instead of exact 9 version

Reply:
I have fixed dependencies so that they now allow for **"greather than"** previously required

Have also used Karma to verify to project still works